packages/go/analysis/ad/adcs.go

```diff
@@ -108,12 +108,44 @@ func PostIssuedSignedBy(ctx context.Context, db graph.Database, enterpriseCertAu
 	return &operation.Stats, operation.Done()
 }
 
+func PostEnterpriseCAFor(ctx context.Context, db graph.Database, enterpriseCertAuthorities []graph.Node) (*analysis.AtomicPostProcessingStats, error) {
+	operation := analysis.NewPostRelationshipOperation(ctx, db, "EnterpriseCAFor Post Processing")
+
+	operation.Operation.SubmitReader(func(ctx context.Context, tx graph.Transaction, outC chan<- analysis.CreatePostRelationshipJob) error {
+		for _, ecaNode := range enterpriseCertAuthorities {
+			if thumbprints, err := ecaNode.Properties.Get(ad.CertThumbprint.String()).StringSlice(); err != nil {
+				return err
+			} else {
+				for _, thumbprint := range thumbprints {
```

Comment:
Good call, that's what I get for copying other code. I also added an empty string check before doing the thumbprint lookup just in case that value is empty for some reason.